### ISSUE TITLE
Block-comment ignore-comments have trailing `*)` in ignore range, preventing match

### ISSUE THREAD

**Issue by Smaug123** [ionide/FSharp.Analyzers.SDK#259]
**Describe the bug**
An ignore line like:

```fsharp
    (* fsharpanalyzer: ignore-line-next WOOF-BLOCKING *)
```
ends up with a `client.AnalyzerIgnoreRanges` with key `WOOF-BLOCKING *)`, not `WOOF-BLOCKING`; this means the ignore doesn't trigger.

It's 1am for me here so I haven't got a clean repro outside WoofWare.FSharpAnalyzers, I'm afraid (I can't actually build the SDK locally for reasons I haven't tried to debug). I think any attempt to reproduce the bug should succeed, though.

**Expected behaviour**
The ignore-line-next directive triggers.

**Screenshots**

**Additional context**
Ref: #254
